core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py

```diff
@@ -110,25 +110,22 @@ def _fit(self, predictions: List[np.ndarray], labels: np.ndarray, time_limit=Non
         round_scores = False
         epsilon = 1e-4
         round_decimals = 6
+        ensemble_prediction = np.zeros(predictions[0].shape)
+        weighted_ensemble_prediction = np.zeros(predictions[0].shape)
+        fant_ensemble_prediction = np.zeros(predictions[0].shape)
```

Comment:
I updated the code a bit to minimize the redundant np.ndarray creations and use in-place operations whenever possible.